zbus/src/connection/builder.rs

```diff
@@ -88,6 +89,22 @@ impl<'a> Builder<'a> {
         Self::address(&crate::address::system()?)
     }
 
+    /// Create a builder for an IBus connection
+    pub fn ibus() -> Result<Self> {
```

Comment:
Most users will not need this (infact until #965 is resolved, we may not have any users of this) so it needs to be feature-gated. Please add a new `ibus` cargo feature and enable this only when that's enabled.

Reply:
What is the major impact if we add feature for this function? Does it only affect the size of the program?

I tried to solve #965 but not quite success.

Reply:
> What is the major impact if we add feature for this function? Does it only affect the size of the program?

Yeah and compilation speed. The impact is small in this case but given that 99% of users won't be using it, best to not enable it by default.